cmd/config/internal/commands/cmdlistsetters.go

```diff
@@ -63,25 +65,41 @@ func (r *ListSettersRunner) preRunE(c *cobra.Command, args []string) error {
 
 func (r *ListSettersRunner) runE(c *cobra.Command, args []string) error {
 	if setterVersion == "v2" {
-		if err := r.ListSetters(c, args); err != nil {
+		openAPIFileName, err := ext.GetOpenAPIFile([]string{"."})
```

Comment:
This seems to mean that the openapi file must be in the current directory. Is this correct?

Reply:
No. We just need the openAPI filename set by upstream (ex: Kptfile). Moved this method into ext to avoid confusion.